src/UBAFeeCalculator/UBAFeeConfig.ts

```diff
@@ -67,6 +86,17 @@ class UBAConfig {
   public getBalancingFeeTuples(chainId: number): TupleParameter[] {
     return this.balancingFee.override?.[chainId] ?? this.balancingFee.default;
   }
+
+  /**
+   * @description Get the balance trigger threshold for a given chain and token
+   * @param chainId The chain id
+   * @param tokenSymbol The token address
+   * @returns The balance trigger threshold if it exists
+   */
+  public getBalanceTriggerThreshold(chainId: number, tokenSymbol: string): TupleParameter | undefined {
```

Comment:
nit: don't we normally use mainnet token address as the unique identifier?

Can't the trigger threshold change over time, so don't we need to specify the block number for which we want to get the threshold?